internal/config/identity/openid/openid.go

```diff
@@ -619,6 +619,13 @@ func GetDefaultExpiration(dsecs string) (time.Duration, error) {
 		}
 
 		defaultExpiryDuration = time.Duration(expirySecs) * time.Second
+	} else if timeout == "" && dsecs == "" {
+		defaultExpiryDuration = time.Duration(60) * time.Minute
+	}
+
+	if defaultExpiryDuration.Seconds() < 900 || defaultExpiryDuration.Seconds() > 31536000 {
```

Comment:
can we make the 900 and 31536000 constants to avoid having [Magic Numbers](https://www.baeldung.com/cs/antipatterns-magic-numbers#magic-numbers). Cause we have the same values repeated above .